### DISCUSSION TITLE
followed people don't show

### DISCUSSION THREAD

**Discussion by kilobyte** [community/community#70236] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

The feed includes plenty of social/AI crap but doesn't even show stuff from people you **explicitly** follow.  Here, I've been molesting a friend to make a piece of software she wrote public.  So here's a repository, https://github.com/glitzerai/sbuild, pushed to **~10 minutes ago**, out of only 3 people I even follow.  And it doesn't even show up in the feed.

Neither the repo nor the person is popular by whatever Facebook-like social metric, but as something I explicitly marked to follow, it should show on the dashboard, prominently at the top, bolded or whatever.  Instead, I see plenty of completely unrelated stuff I don't even see a way to disable.

The dashboard in its present state is useless.

**Comment by segabl** [comment]
yeah it's awful not seeing activities by people you follow. I dont want to go to all their repos manually to watch them just to get updates (in a more annoying way than previously)
